app/src/main/java/com/getsensibill/spendmanager/demo/capture_flow/CaptureFlowJavaActivity.java

```diff
@@ -0,0 +1,84 @@
+package com.getsensibill.spendmanager.demo.capture_flow;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.view.View;
+
+import com.getsensibill.captureflow.coordinator.CaptureFlowCoordinator;
+import com.getsensibill.captureflow.coordinator.CaptureFlowState;
+import com.getsensibill.capturestandalone.models.CaptureConfig;
+import com.getsensibill.core.analytic.Transaction;
+import com.getsensibill.spendmanager.demo.databinding.ActivityCaptureFlowJavaBinding;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Date;
+
+/**
+ * Capture Flow Integration
+ *
+ * This is an example on how you can track the state changing during a capture flow of a receipt
+ * using the [CaptureFlowCoordinator].
+ */
+public class CaptureFlowJavaActivity extends AppCompatActivity {
+    private ActivityCaptureFlowJavaBinding binding;
+
+    // Creates a capture flow Coordinator
+    private CaptureFlowCoordinator captureFlow = new CaptureFlowCoordinator(this);
+
+    // Setup the capture flow listener which is used to listen to the state changing
+    private CaptureFlowCoordinator.CaptureFlowListener captureFlowListener =
+            new CaptureFlowCoordinator.CaptureFlowListener() {
```

Comment:
Indentation looks a little weird here, is it as intended?  Maybe we should one-line this

Reply:
updated. it was new lined as its long, but probably still easier to read on one line